### ISSUE TITLE
:inotifywait.find_executable/0 is undefined

### ISSUE THREAD

**Issue by formido** [falood/exsync#5]
I followed the instructions in the readme, ran `mix deps.get`, but when I run `iex -S mix`, I get the following:

```
    ** (EXIT) an exception was raised:
        ** (UndefinedFunctionError) function :inotifywait.find_executable/0 is undefined (module :inotifywait is not available)
            :inotifywait.find_executable()
            (exfswatch) lib/exfswatch.ex:19: ExFSWatch.start/2
            (kernel) application_master.erl:273: :application_master.start_it_old/4
```

**Comment by formido** [comment]
When I run `mix deps.get`, it's pulling in `exfswatch` but not `fs`.

**Comment by formido** [comment]
Looks like it's due to this:

http://stackoverflow.com/questions/38730056/why-doesnt-mix-pull-this-transitive-dependency/38730291#38730291

...and I see you've already filed synrc/fs#24, so I don't know. Maybe a little explanation in the README or switching the dependencies to git?

**Comment by falood** [comment]
I have released `v0.2.1` of `exfswatch` to fixed it and also released new version for `exsync`.
Thank you.
